examples/strings/strings.bal

```diff
@@ -64,18 +64,20 @@ public function main() {
     boolean contains = statement.includes("Town", 7);
     io:println("Contains: ", contains);
 
-    // Format a `string` according to the given format arguments.
+    // Format a `string` according to the given format arguments
+    // using string templates.
     string name = "Sam";
-    int marks = 90;
+    int mrk = 90;
```

Comment:
```suggestion
    int marks = 90;
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1442/commits/cd8c2d975eb65d4905cb6169f946b82be90e5204